bindings/go/README.md

```diff
@@ -1,20 +1,20 @@
-# Go bindings
+# Go Bindings for the C-KZG Library
 
-This package implements Go bindings (using [Cgo](https://go.dev/blog/cgo)) for the
-exported functions in [C-KZG-4844](https://github.com/ethereum/c-kzg-4844).
+This directory contains Go bindings for the C-KZG-4844 library.
+The bindings use [Cgo](https://go.dev/blog/cgo).
```

Comment:
I think it would be okay to delete the remark about Cgo. I added this a while back & it's not really that useful.